sources/packages/backend/workflow/test/2023-2024/parttime-assessment/awards-amounts/parttime-assessment-awards-amount-CSGP.e2e-spec.ts

```diff
@@ -0,0 +1,50 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CSGP.`, () => {
+  it("Should determine federalAwardCSGPAmount when awardEligibilityCSGP is true when studentDataApplicationPDPPDStatus is true", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.programYearTotalCSGP).toBe(130);
+    expect(calculatedAssessment.variables.federalAwardNetCSGPAmount).toBe(2670);
+    expect(calculatedAssessment.variables.finalFederalAwardNetCSGPAmount).toBe(
+      2670,
+    );
+  });
+
+  it("Should determine federalAwardNetCSGPAmount as zero when awardEligibilityCSGP is true and programYearTotalCSGP (programYearTotalFullTimeCSGP + programYearTotalPartTimeCSGP) is greater than the dmnPartTimeAwardAllowableLimits for CSGP ", async () => {
```

Comment:
I think we should avoid adding "(programYearTotalFullTimeCSGP + programYearTotalPartTimeCSGP)" to the test name as programYearTotalCSGP is enough and the logic to calculate programYearTotalCSGP might change.